src/components/index.ts

```diff
@@ -2,6 +2,7 @@ export * from "./BoxLabel"
 export * from "./Card"
 export * from "./ChecklistGroup"
 export * from "./ChecklistItem"
+export * from "./Countdown"
```

Comment:
Maybe it's worth exporting `utils` functions as well.

Reply:
Fixed in f9fb299b2fb9f11f1b05ecd91a22047d8198686a